src/Components/Publishing/Header/__test__/__snapshots__/FeatureHeader.test.tsx.snap

```diff
@@ -2902,8 +2907,9 @@ exports[`feature renders superArticle full header properly 1`] = `
 }
 
 .c0[data-type="split"] .c13 {
+  -webkit-box-flex: 1;
   -webkit-flex-grow: 1;
-  -ms-flex-grow: 1;
+  -ms-flex-positive: 1;
```

Comment:
Any information about these changes? /cc @kana @eessex

Reply:
This looks correct according to https://msdn.microsoft.com/en-us/library/hh673531(v=vs.85).aspx. Maybe some dependency that got upgraded fixed the processor to spit out the right -ms prefix for `flex-grow`?

Reply:
Probably yeah 👍